### ISSUE TITLE
outdated jetty version leads to SSL_ERROR_NO_CYPHER_OVERLAP

### ISSUE THREAD

**Issue by q-jack** [forcedotcom/SalesforceCanvasFrameworkSDK#26]
In the pom there is a version of jetty which is already EOL (https://wiki.eclipse.org/Jetty/Howto/CipherSuites)
If I replace the webapp with a new version (e.g. 9.4.35.v20201120) I get the following error when starting jetty.
![image](https://user-images.githubusercontent.com/11267539/101332521-b42b5580-3875-11eb-9541-254c1102bfb4.png)

I'm trying to use the example of the Canvas app on https://developer.salesforce.com/docs/atlas.en-us.platform_connect.meta/platform_connect/quick_start_run_the_app_locally.htm. At the point "Run the web App locally" I get the error message "SSL_ERROR_NO_CYPHER_OVERLAP".

**Comment by Raneet10** [comment]
Yeah, the same is happening with me too. But do you get the  "SSL_ERROR_NO_CYPHER_OVERLAP" message when navigating to `https://localhost:8443/examples/hello-world/index.jsp` ? In my case,the webapp stops after giving the message in the screenshot.

**Comment by q-jack** [comment]
I get the error msg when navigating to https://localhost:8443/examples/hello-world/index.jsp. I get the error msg in firefox. 
In chrome I get the msg ERR_SSL_PROTOCOL_ERROR
Starting webapp looks good to me (with the version(7.6.0.v20120127) of jetty from repository):
![image](https://user-images.githubusercontent.com/11267539/101448329-1f813000-3927-11eb-8547-6471f3dd2a4d.png)

**Comment by Raneet10** [comment]
![image](https://user-images.githubusercontent.com/35629432/101494390-48d4a700-398d-11eb-83e8-de13f2110671.png)
I come across this when using the older jetty version.
